projects/storefrontapp/src/app/qualtrics-extended.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@angular/core';
+import {
+  CartService,
+  QualtricsConfig,
+  QualtricsConfigService,
+  WindowRef,
+} from '@spartacus/core';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+
+@Injectable({ providedIn: 'root' })
+export class QualtricsExtendedService extends QualtricsConfigService {
```

Comment:
This is an example, right? Shall we rename it to `DemoQualtricsLoader`

Reply:
yup, its an example, so it won't be put in the final merge! just for testing purposes, therefore no need to change the name :p